### PR TITLE
A11y: New animation for live tag [KDS-1523]

### PR DESCRIPTION
**Pull request by christian314159**
## Why
- Fixes cultureamp/kaizen-discourse#91

## What
- Replaced the pulse animation with an icon that cycles through 3 animation loops and then stops

image

### CODE REVIEW

**Review by gyfchong — DISMISSED** [review]

**Comment by jmitsios** [comment]
@christian314159 — I noticed a little rough and jagged edges around the icon after a moment of looking at it — so I zoomed in to check it out:
image

Are the 'lighter' lines remaining in position when the darker ones appear over the top of them, or are they disappearing? Is this double-up of shapes causing a harder fuzzy edge rather than a sharp and smooth one?

File: draft-packages/tag/docs/Tag.stories.tsx [hunk 1]
```diff
@@ -164,3 +164,9 @@ StickerSheetReversed.parameters = {
   chromatic: { disable: false },
   controls: { disable: true },
 }
+
+export const LiveTag: StoryFn<typeof Tag> = args => (
```
Comment (dougmacknz) [inline draft-packages/tag/docs/Tag.stories.tsx:168]: Do we definitely want another story for this? It's covered in the sticker sheet

File: .changeset/chilly-coats-vanish.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@kaizen/draft-tag": minor
```
Comment (dougmacknz) [inline .changeset/chilly-coats-vanish.md:2]: Do we need a bump for component library as well for the icon being added?

Comment (christian314159) [reply .changeset/chilly-coats-vanish.md:2]: Yes, you're right, thanks!

File: draft-packages/tag/docs/Tag.stories.tsx (hunk 1 above)
Comment (christian314159) [reply draft-packages/tag/docs/Tag.stories.tsx:168]: No, we don't. Added it just for dev convenience.

**Review by gyfchong — APPROVED** [review]

**Pull request merged**